flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/ServerTransportErrorHandlingTest.java

```diff
@@ -101,7 +104,22 @@ public void channelRead(ChannelHandlerContext ctx, Object msg)
         NettyTestUtil.NettyServerAndClient serverAndClient = null;
 
         try {
-            serverAndClient = initServerAndClient(protocol, createConfig());
+            for (int retry = 0; retry < NETTY_INIT_MAX_RETRY_TIMES; retry++) {
+                try {
+                    serverAndClient = initServerAndClient(protocol, createConfig());
+                    break;
+                } catch (Exception e) {
+                    if (retry >= NETTY_INIT_MAX_RETRY_TIMES - 1) {
+                        throw e;
+                    }
+                    if (e instanceof BindException
```

Comment:
I am not sure but from the test url I find the `Exception` is following
![image](https://user-images.githubusercontent.com/26054913/139632551-4fcabc48-b11f-4d95-93c9-8ef6804100b5.png)

So I would suggest to see what exactly exception is when the port is used.
WDYT?

Reply:
Thanks. I will make the exception inevitable by init the server using an occupied port. And I will print the detailed stack of the exception by triggering it in the CI test.